helm/ds/values.yaml

```diff
@@ -69,14 +69,18 @@ backup:
 #opendjJavaArgs: "-server -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:+UseCompressedOops -XX:+UseG1GC -XX:MaxGCPauseMillis=100 -XX:MaxRAMFraction=2"
 
 opendjJavaArgs: "-Xmx1024m -XX:+UnlockDiagnosticVMOptions"
+
 # Resource limits.
-# These help for pod placement on a larger cluster to ensure the OpenDJ instance gets sufficient resources.
-# For production you will want to specify CPU limits to help Kube schedule the pods.
-resources:
- requests:
-  memory: 2000Mi
- limits:
-   memory: 2500Mi
+# These help for pod placement in a larger cluster to ensure the DS instance gets sufficient resources.
+# For production, you will want to specify CPU limits to help Kube schedule the pods.
+resources: {}
+
+# Sample values:
+#resources:
```

Comment:
How is this different?

Reply:
In the original, we have:

resources:
 requests:
  memory: 2000Mi
 limits:
   memory: 2500Mi

Here I have

resources: {}

(which is what's in the opendj Helm chart)

I was assuming this defaulted to something low but maybe not; do I need to specify an explicit memory amount? (If so, what's the lowest we can go?)

Reply:
we ran into issues with this before during training.  It would be better to set lower limits (and also adjust java opts down)